thunder/dynamo/benchmark_utils.py

```diff
@@ -375,8 +385,28 @@ def check_timing(
     and generate a benchmark script if the difference exceeds the threshold.
     """
     graph_name = report.graph_name
-    measure1 = report.run_benchmark(compile_fn1, timer_fn)
-    measure2 = report.run_benchmark(compile_fn2, timer_fn)
+    if not is_report_using_cuda(report):
```

Comment:
This is totally fine. In the future we may want to make it an option to skip benchmarking when CUDA isn't being used.